src/EnergyPlus/HVACFan.cc

```diff
@@ -305,15 +310,67 @@ namespace HVACFan {
             OutputReportPredefined::PreDefTableEntry(OutputReportPredefined::pdchFanPwrPerFlow, name, designElecPower / designAirVolFlowRate);
         }
         OutputReportPredefined::PreDefTableEntry(OutputReportPredefined::pdchFanMotorIn, name, m_motorInAirFrac);
+        if (m_reportFEIFlag) {
+            OutputReportPredefined::PreDefTableEntry(OutputReportPredefined::pdchFanEnergyIndex, name, m_designPointFEI);
+        }
+
         OutputReportPredefined::PreDefTableEntry(OutputReportPredefined::pdchFanEndUse, name, m_endUseSubcategoryName);
 
         m_objSizingFlag = false;
     }
 
+    void FanSystem::report_fei()
+    {
+        // PURPOSE OF THIS SUBROUTINE:
+        // Calculate the Fan Energy Index
+
+        // REFERENCES:
+        // ANSI/AMCA Standard 207-17: Fan System Efficiency and Fan System Input Power Calculation, 2017.
+        // AANSI / AMCA Standard 208 - 18: Calculation of the Fan Energy Index, 2018.
+
+        // Using/Aliasing
+        using Psychrometrics::PsyRhoAirFnPbTdbW;
+
+        // SUBROUTINE PARAMETER DEFINITIONS:
+        Real64 RhoAir;
```

Comment:
These variables don't need to be declared at the top of the function anymore.  Declare them and use them right where they are used, and use `const` where possible to allow the compiler to optimize as much as possible.